### PR TITLE
docs(sqlite3): remove references to vscode fork

### LINKED ISSUE #83: Remove references to vscode fork of sqlite3
Work is being done on https://github.com/TryGhost/node-sqlite3 again (the repo has been transferred to another organization). I expect that a new release will be out soon and then we should try to move back to the original package and remove the warnings in our docs.

### PR DESCRIPTION
**Pull request by WikiRik**
The vscode fork is not needed anymore now that sqlite3 is being updated again.
Relates to https://github.com/sequelize/sequelize/pull/14384
Closes #83

### CODE REVIEW

**Review by fzn0x — APPROVED** [review]

File: src/pages/releases.md [hunk 1]
```diff
@@ -85,31 +85,12 @@ MSSQL requires the use of the [tedious] npm package.
...
-  }
-}
-```
-
-:::
-
 | Sequelize   | [sqlite3]                                                      |
 |-------------|----------------------------------------------------------------|
-| 7 (alpha)   | `npm:@vscode/sqlite3@^5.0.7`                                   |
-| 6 (current) | `npm:@vscode/sqlite3@^4.0.12`, or `npm:@vscode/sqlite3@^5.0.7` |
```
Comment (ephys) [inline src/pages/releases.md:110]: `sqlite3@^4.0.12` only exists on the vscode fork, the new sqlite3 release is only applicable to 5.x.x

Comment (WikiRik) [reply src/pages/releases.md:110]: Shall we make it 4.0.9? https://www.npmjs.com/package/sqlite3
Or why did you use 4.0.12?

Comment (ephys) [reply src/pages/releases.md:110]: It's the release that fixed the issues with the 4.x branch. We can declare we support `>4.0.0` but that using either `>=5.0.3` or `npm:@vscode/sqlite3@^4.0.12` is recommended?

Comment (WikiRik) [reply src/pages/releases.md:110]: Yes, I'll update the v6 docs and this table to reflect this later today

**Review by ephys — APPROVED** [review]

**Pull request merged**